catalog.html

```diff
@@ -8,98 +8,129 @@
   <link href="css/style.css" rel="stylesheet">
 </head>
 <body>
-  <header class="page-header">
-    <a href="index.html">
-      <img src="img/logo-sedona.svg" width="140" height="70" alt="Логотип">
-    </a>
-    <nav>
-      <ul class="site-navigation">
-        <li><a href="#">Информация</a></li>
-        <li><a href="#">Фото и видео</a></li>
-        <li><a href="#">Карта штата</a></li>
-        <li><a class="active-link" href="#">Гостиницы</a></li>
+  <div class="wrapper">
```

Comment:
Я предлагаю использовать общепринятое название класса `container` для таких случаев:
```suggestion
  <div class="container">
```